packages/auth/package.json

```diff
@@ -21,12 +21,25 @@
     "test": "jest src",
     "test:watch": "yarn test --watch"
   },
+  "peerDependencies": {
+    "@clerk/clerk-react": "3.2.8",
+    "@clerk/clerk-sdk-node": "3.3.6"
+  },
+  "peerDependenciesMeta": {
```

Comment:
Cool! First time I see one of these out in the wild 🙂 Hadn't even heard about them until the whole React 18 types debacle

Reply:
Ha I copied it from the api/package.json - I also hadn't seen it in the wild yet!